packages/mirage-flow-rawlink/mirage-flow-rawlink.1.0.0/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "thomas@gazagnaire.org"
+authors: ["Thomas Gazagnaire"]
+license: "ISC"
+homepage: "https://github.com/mirage/mirage-flow-rawlink"
+dev-repo: "https://github.com/mirage/mirage-flow-rawlink.git"
+bug-reports: "https://github.com/mirage/mirage-flow-rawlink/issues"
+doc: "https://mirage.github.io/mirage-flow-rawlink/"
+
+build: [ "jbuilder" "build" "-p" name "-j" jobs ]
+
+depends: [
+  "jbuilder" {build & >="1.0+beta10"}
+  "mirage-flow-lwt" {>= "1.2.0"}
+  "rawlink" {>= "0.5"}
+  "cstruct"
```

Comment:
This should be >= 3.0.2 now, as testing the older ones with jbuilder is quite hard